XML3-Homework/hering_03.xml

```diff
@@ -0,0 +1,258 @@
+        <line num="147">This is not good, and I know <rhy>why</rhy>.</line>
+        <line num="148">A mouse has cut the wire, <rhy>goodbye</rhy>!</line>
+    </section>
+
+    <section num="21" rhySchemes="11">
+        <line num="149">From <rhy>near</rhy> to far, from <rhy>here</rhy> to <rhy>there</rhy>,</line>
+        <line num="150">Funny things are <rhy>everywhere</rhy>.</line>
+        <line num="151">These <color>yellow</color> pets are called the <rhy>Zeds</rhy>.</line>
+        <line num="152">They have one hair upon their <rhy>heads</rhy>.</line>
+        <line num="153">Their hair grows fast. So fast they <rhy>say</rhy>,</line>
+        <line num="154">They need a haircut every <rhy>day</rhy>.</line>
+    </section>
+
+    <section num="22" rhySchemes="11">
+        <line num="155">Who am I? My name is <rhy>Ish</rhy></line>
+        <line num="156">On my hand I have a <rhy>dish</rhy>.</line>
+        <line num="157">I have this <rhy>dish</rhy> to help me <rhy>wish</rhy>.</line>
+        <line num="158">When I <rhy>wish</rhy> to make a <rhy>wish</rhy></line>
+        <line num="159">I wave my hand with a big <rhy>swish</rhy> <rhy>swish</rhy>.</line>
+        <line num="160">Then I say, "I <rhy>wish</rhy> for <rhy>fish</rhy>!"</line>
+        <line num="161">And I get <rhy>fish</rhy> right on my <rhy>dish</rhy>.</line>
+        <line num="162">So...</line>
+        <line num="163">If you <rhy>wish</rhy> to make a <rhy>wish</rhy>,</line>
+        <line num="164">you may <rhy>swish</rhy> for <rhy>fish</rhy> with my <rhy>Ish</rhy> <rhy>wish</rhy> <rhy>dish</rhy>.</line>
+    </section>
+
+    <section num="23" rhySchemes="11">
+        <line num="165">At our house we play out <rhy>back</rhy>.</line>
+        <line num="166">We play a game called ring the <rhy>Gack</rhy>.</line>
+        <line num="167">Would you like to play this game?</line>
+        <line num="168">Come down! We have the only <rhy>Gack</rhy> in town.</line>
+    </section>
+
+    <section num="24" rhySchemes="11">
+        <line num="169">Look what we found in the park in the <rhy>dark</rhy>.</line>
+        <line num="170">We will take him home, we will call him <rhy>Clark</rhy>.</line>
+        <line num="171">He will live at our house, he will <rhy>grow</rhy> and <rhy>grow</rhy>.</line>
+        <line num="172">Will our mother like this? We don't <rhy>know</rhy>.</line>
+    </section>
+
+    <section num="25" rhySchemes="11">
+        <line num="173">And now, Good night.</line>
+        <line num="174">It is time to <rhy>sleep</rhy></line>
+        <line num="175">So we will <rhy>sleep</rhy> with our pet <rhy>Zeep</rhy>.</line>
+        <line num="176">Today is gone. Today was <rhy>fun</rhy>.</line>
+        <line num="177">Tomorrow is another <rhy>one</rhy>.</line>
+        <line num="178">Every day, from here to <rhy>there</rhy>.</line>
+        <line num="179">funny things are <rhy>everywhere</rhy>.</line>
```

Comment:
@luh429 As you think about coding rhyme schemes, maybe consider having the `<rhy>` markup wrap around just the portion of the word that sounds the rhyme! That may look funny but it's perfectly fine from the point of view of XML syntax. So here, we could try: 

```xml
 <line num="178">Every day, from here to <rhy>there</rhy>.</line>
        <line num="179">funny things are every<rhy>where</rhy>.</line>
```

Reply:
@luh429 What I'm demonstrating here is **line-commenting**, which you can do on any commit pushed to a repo. We commonly do this on Pull Requests because GitHub makes it easy to comment on particular lines of code and start a conversation about them. You can test out replying to one of these comments, for example.